Build fails with Qt >= 6.10

A change in Qt internal #includes appears to have made `QElapsedTimer` no longer visible in `sacneffectengine.h`:

```
/home/dankeenan/Documents/Projects/sacnview/src/sacn/sacneffectengine.h:124:5: error: ‘QElapsedTimer’ does not name a type
  124 |     QElapsedTimer m_dwellTimeTracker;
      |     ^~~~~~~~~~~~~
```

--- a/src/sacn/sacneffectengine.h
+++ b/src/sacn/sacneffectengine.h
@@ -17,6 +17,7 @@
 #define SACNEFFECTENGINE_H
 
 #include "sacnsender.h"
+#include <QElapsedTimer>
 #include <QImage>
 #include <QObject>
 #include <QPixmap>

